src/css/base.scss

```diff
@@ -336,3 +337,7 @@
 .hide {
   display: none;
 }
+
+.hidden {
+  display: none;
+}
```

Comment:
two hide classes are unnecessary (if you're relying on one for a certain thing like a filter, try giving it a more semantic name)

Reply:
Okay took out the hide class